modules/test/playwright/tests/analytics-web/supportLiferayCookieManager.spec.ts

```diff
@@ -32,7 +32,7 @@ export const test = mergeTests(
 	loginTest()
 );
 
-async function changeCookiePreference(page, options) {
+async function changeCookiePreference(page: Page, options) {
```

Comment:
@ViniLopes20 , can we define the type of `options` parameter?

Reply:
Done